README.md

```diff
@@ -133,31 +133,31 @@ for index in range(0, 10):
+GillesPy2's [online documentation](https://stochss.github.io/GillesPy2/) provides more details about using the software.  If you find any problem with GillesPy2 or the documentation, please report it using the [GitHub issue tracker](https://github.com/StochSS/GillesPy2/issues) for this repository.  You can also contact Dr. [Brian Drawert](http://www.cs.unca.edu/~drawert) directly with questions and suggestions.
 
 Contributing
 ------------
 
-We would be happy to receive your help and participation with enhancing GillesPy2!  The [UML class diagram](UML_CLASS_DIAGRAM.md) and [Pynsource](https://pynsource.com/) [UML class model](docs/getting_started/basic_usage/gillespy2-UML-class-model.pyns) may help you familiarize yourself with the existing code. Please follow the guidelines described in [CONTRIBUTING.md](https://github.com/GillesPy2/GillesPy2/tree/main/CONTRIBUTING.md).
+We would be happy to receive your help and participation with enhancing GillesPy2!  The [UML class diagram](UML_CLASS_DIAGRAM.md) and [Pynsource](https://pynsource.com/) [UML class model](docs/getting_started/basic_usage/gillespy2-UML-class-model.pyns) may help you familiarize yourself with the existing code. Please follow the guidelines described in [CONTRIBUTING.md](https://github.com/StochSS/GillesPy2/tree/main/CONTRIBUTING.md).
 
-New developments happen primarily in the [`develop`](https://github.com/GillesPy2/GillesPy2/commits/develop) branch.  New releases are put in the `main` branch.
+New developments happen primarily in the [`develop`](https://github.com/StochSS/GillesPy2/commits/develop) branch.  New releases are put in the `main` branch.
 
 <p align="center">
 
 | Main Branch   | Develop Branch | Test Coverage | Maintainability |
 |:---------------:|:--------------:|:--------:|:---------------:|
-| [![Build Status](https://travis-ci.org/GillesPy2/GillesPy2.svg?branch=main)](https://travis-ci.org/GillesPy2/GillesPy2) | [![Build Status](https://travis-ci.org/GillesPy2/GillesPy2.svg?branch=develop)](https://travis-ci.org/GillesPy2/GillesPy2) | [![Test Coverage](https://api.codeclimate.com/v1/badges/990ac9d778d681d32eea/test_coverage)](https://codeclimate.com/github/GillesPy2/GillesPy2/test_coverage) | [![Maintainability](https://api.codeclimate.com/v1/badges/990ac9d778d681d32eea/maintainability)](https://codeclimate.com/github/GillesPy2/GillesPy2/maintainability) |
+| [![Build Status](https://travis-ci.org/StochSS/GillesPy2.svg?branch=main)](https://travis-ci.org/StochSS/GillesPy2) | [![Build Status](https://travis-ci.org/StochSS/GillesPy2.svg?branch=develop)](https://travis-ci.org/StochSS/GillesPy2) | [![Test Coverage](https://api.codeclimate.com/v1/badges/990ac9d778d681d32eea/test_coverage)](https://codeclimate.com/github/StochSS/GillesPy2/test_coverage) | [![Maintainability](https://api.codeclimate.com/v1/badges/990ac9d778d681d32eea/maintainability)](https://codeclimate.com/github/StochSS/GillesPy2/maintainability) |
```

Comment:
The Code Climate badges still work using the GillesPy2/GillesPy2 links, not StochSS/GillesPy2.